src/Compilers/CSharp/Test/Semantic/Semantics/ConstantTests.cs

```diff
@@ -2443,6 +2443,324 @@ void M()
             Assert.Equal(expected, actual);
         }
 
+        [Fact]
+        public void TestUnspecifiedShifts()
```

Comment:
>TestUnspecifiedShifts [](start = 20, length = 21)

Does VB have the same problem? Please add a test.

Reply:
VB already has masking for its shift operators and tests.